src/utils/Processing.ts

```diff
@@ -240,6 +241,46 @@ export function fillTank(rawFSHes: RawFSH[], config: Configuration): FSHTank {
   return new FSHTank(docs, config);
 }
 
+export function checkNullValuesOnArray(resource: any, parentName = '', priorPath = ''): void {
+  const resourceName = parentName ? parentName : resource.id ?? resource.name;
```

Comment:
Oh this is one other really small code styling change, but I think you can chain the `??` together, so I think this line could be `parentName ?? resource.id ?? resource.name`. I'm not sure if there's any other benefit besides it looking simpler, though.

Reply:
I tried switching this line over to chaining the `??`s together but that didn't work. I think this is because the ternary operator checks if `parentName` is truthy but the `??` operator only checks if it is defined, which it always will be. Because of that I think it'd be best to leave this line as is unless you think it'd be best for me rework my approach.

Reply:
Ohh the truthy piece of this is definitely subtle and real and I missed it, so I'm good with keeping it how you have it.